xml/System/Byte.xml

```diff
@@ -1,6960 +1,6995 @@
-      <MemberType>Method</MemberType>
-      <Implements>
-        <InterfaceMember>M:System.Numerics.IBinaryInteger`1.TryWriteBigEndian(System.Span{System.Byte},System.Int32@)</InterfaceMember>
-      </Implements>
-      <AssemblyInfo>
-        <AssemblyName>System.Runtime</AssemblyName>
-        <AssemblyVersion>7.0.0.0</AssemblyVersion>
-        <AssemblyVersion>8.0.0.0</AssemblyVersion>
-      </AssemblyInfo>
-      <AssemblyInfo>
-        <AssemblyName>mscorlib</AssemblyName>
-      </AssemblyInfo>
-      <AssemblyInfo>
-        <AssemblyName>netstandard</AssemblyName>
-      </AssemblyInfo>
-      <ReturnValue>
-        <ReturnType>System.Boolean</ReturnType>
-      </ReturnValue>
-      <Parameters>
-        <Parameter Name="destination" Type="System.Span&lt;System.Byte&gt;" Index="0" FrameworkAlternate="net-7.0;net-8.0" />
-        <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="net-7.0;net-8.0" />
-      </Parameters>
-      <Docs>
-        <param name="destination">The span to which the current value should be written.</param>
-        <param name="bytesWritten">When this method returns, contains the number of bytes written to <paramref name="destination" />.</param>
-        <summary>Tries to write the current value, in big-endian format, to a given span.</summary>
-        <returns>
-          <see langword="true" /> if the value was succesfully written to <paramref name="destination" />; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
-        <inheritdoc cref="M:System.Numerics.IBinaryInteger`1.TryWriteBigEndian(System.Span{System.Byte},System.Int32@)" />
-      </Docs>
-    </Member>
-    <Member MemberName="System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteLittleEndian">
-      <MemberSignature Language="C#" Value="bool IBinaryInteger&lt;byte&gt;.TryWriteLittleEndian (Span&lt;byte&gt; destination, out int bytesWritten);" />
-      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance bool System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteLittleEndian(valuetype System.Span`1&lt;unsigned int8&gt; destination, [out] int32&amp; bytesWritten) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.Byte.System#Numerics#IBinaryInteger&lt;System#Byte&gt;#TryWriteLittleEndian(System.Span{System.Byte},System.Int32@)" />
-      <MemberSignature Language="VB.NET" Value="Function TryWriteLittleEndian (destination As Span(Of Byte), ByRef bytesWritten As Integer) As Boolean Implements IBinaryInteger(Of Byte).TryWriteLittleEndian" />
-      <MemberSignature Language="F#" Value="abstract member System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteLittleEndian : Span&lt;byte&gt; * int -&gt; bool&#xA;override this.System.Numerics.IBinaryInteger&lt;System.Byte&gt;.TryWriteLittleEndian : Span&lt;byte&gt; * int -&gt; bool" Usage="byte.System.Numerics.I
```

Comment:
Shouldn't these ones be `inheritdoc`?

Reply:
Hmmm, I think this flagged the wrong lines...

The ask was for `L3269-L3272`: https://github.com/dotnet/dotnet-api-docs/pull/9187/files#diff-3432525d875edc26d3d7670c8a607158333268678711ffe1225e253a9f8b7658R3269-R3272:
```csharp
        <param name="left">The value to which <paramref name="right" /> is added.</param>
        <param name="right">The value which is added to <paramref name="left" />.</param>
        <summary>Adds two values together to compute their sum.</summary>
        <returns>The sum of <paramref name="left" /> and <paramref name="right" />.</returns>
```

Most of these new comments are `inheritdoc` in source and so I'd expect inheritdoc here as well?

Reply:
I ran the tool with inheritdoc check enabled. Let me see...

Reply:
For reference: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Byte.cs#L261-L265

Reply:
Ok then yes, inheritdoc is not ported literally. Instead, the tool finds the interface and copies the documentation from the interface.

Reply:
Here it is: https://github.com/dotnet/runtime/blob/58e03495cf2385f7104acb017df621984d4b7036/src/libraries/System.Private.CoreLib/src/System/Numerics/IAdditionOperators.cs